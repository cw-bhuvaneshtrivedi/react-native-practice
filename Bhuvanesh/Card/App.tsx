import React from "react";
import { StatusBar } from "expo-status-bar";
import { View } from "react-native";
import globalStyles from "./Styles";
import { Card } from "./components/Card";
import { FlashList } from "@shopify/flash-list";
import tmp from "./tmp.json";

export default function App() {
  return (
    <React.StrictMode>
      <View style={globalStyles.body}>
        <StatusBar />
        <FlashList
          data={tmp.models}
          estimatedItemSize={360}
          renderItem={({ item }) => <Card carData={item} />}
        />
        {/* <Card /> */}
      </View>
    </React.StrictMode>
  );
}
