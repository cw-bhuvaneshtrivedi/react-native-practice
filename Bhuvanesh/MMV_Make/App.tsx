import { StatusBar } from "expo-status-bar";
import Home from "./components/Home";
import { useState } from "react";
import { Button, Platform, StyleSheet, View } from "react-native";

export default function App() {
  const [isOpen, setIsOpen] = useState(false);

  return (
    <>
      <StatusBar style="auto" />
      <View style={styles.body}>
        {isOpen && <Home setOpen={setIsOpen} />}
        <Button title="Open" onPress={() => setIsOpen(true)} testID="Open" />
      </View>
    </>
  );
}

const styles = StyleSheet.create({
  body: {
    backgroundColor: "black",
    paddingTop: Platform.OS === "android" ? 24 : 0,
    // justifyContent: "center",
  },
});
