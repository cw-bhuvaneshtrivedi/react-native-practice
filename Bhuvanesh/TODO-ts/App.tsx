import { useState } from "react";
import { Button, FlatList, Text, TextInput, View } from "react-native";

import { Todo } from "./components/todo";
import styles from "./Styles";
import colors from "./constants/colors";

export default function App() {
  const [input, setInput] = useState("");
  const [tasks, setTasks] = useState([
    { id: "1", task: "Create a TODO App" },
    { id: "2", task: "View YT video" },
  ]);

  return (
    <View style={styles.body}>
      <Text style={styles.title}>My Todos</Text>
      <View style={styles.rowFlex}>
        <TextInput
          value={input}
          style={styles.input}
          onChangeText={setInput}
          placeholder="Add Something to do ..."
        />
        <Button
          color={colors.blue}
          title="Save"
          onPress={() => {
            if (input == "") {
              alert("Cannot add empty todo");
              return;
            }
            setTasks((tasks) => [
              ...tasks,
              { id: Date.now().toString(), task: input },
            ]);
            setInput("");
          }}
        />
      </View>
      {tasks.length > 0 ? (
        <FlatList
          data={tasks}
          keyExtractor={(item) => item.id}
          renderItem={({ item }) => (
            <Todo
              id={item.id}
              todo={item.task}
              deleteTodo={() => {
                setTasks(tasks.filter((task) => task.id !== item.id));
                alert("Todo deleted");
              }}
            />
          )}
        />
      ) : (
        <Text>No todos left type to add</Text>
      )}
    </View>
  );
}
