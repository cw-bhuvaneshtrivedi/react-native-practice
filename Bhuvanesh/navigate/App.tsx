import { Provider } from "react-redux";
import { NavigationContainer } from "@react-navigation/native";

import store from "./store";
import { StatusBar } from "react-native";
import Stack from "./routes/homeStack";
import { DrawerContainer } from "./routes/DrawerContainer";
import { Stack2 } from "./components/Stack2";
import { Stack1 } from "./components/Stack1";
import { Stack3 } from "./components/Stack3";

export default function App() {
  return (
    <Provider store={store}>
      <StatusBar />
      <NavigationContainer>
        <Stack.Navigator
          screenOptions={{
            headerTintColor: "white",
            headerStyle: { backgroundColor: "tomato" },
          }}
        >
          <Stack.Screen
            options={{ headerShown: false }}
            name="Drawer"
            component={DrawerContainer}
          />
          <Stack.Screen name="Stack1" component={Stack1} />
          <Stack.Screen name="Stack2" component={Stack2} />
          <Stack.Screen name="Stack3" component={Stack3} />
        </Stack.Navigator>
      </NavigationContainer>
    </Provider>
  );
}
