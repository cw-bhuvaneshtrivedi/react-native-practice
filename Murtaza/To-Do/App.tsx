import React, { useState } from "react";
import {
  StyleSheet,
  View,
  Text,
  TextInput,
  TouchableOpacity,
  FlatList,
  StatusBar,
  Platform
} from "react-native";

type Todo = {
  id: string;
  text: string;
};

const App = () => {
  const [task, setTask] = useState<string>("");
  const [todos, setTodos] = useState<Todo[]>([]);

  const addTodo = () => {
    if (task.trim() === "") return;

    const newTodo: Todo = {
      id: new Date().getTime().toString(),
      text: task,
    };

    setTodos([...todos, newTodo]);
    setTask("");
  };

  const removeTodo = (id: string) => {
    const updatedTodos = todos.filter((todo) => todo.id !== id);
    setTodos(updatedTodos);
  };

  const renderTodoItem = ({ item }: { item: Todo }) => (
    <View style={styles.todoItem}>
      <Text>{item.text}</Text>
      <TouchableOpacity onPress={() => removeTodo(item.id)}>
        <Text style={styles.deleteButton}>Delete</Text>
      </TouchableOpacity>
    </View>
  );

  return (
    <View style={styles.container}>
      <Text style={styles.title}>Todo App</Text>
      <TextInput
        style={styles.input}
        placeholder="Enter your task"
        value={task}
        onChangeText={(e) => setTask(e)}
      />
      <TouchableOpacity style={styles.addButton} onPress={addTodo}>
        <Text style={styles.addButtonText}>Add Task</Text>
      </TouchableOpacity>
      <FlatList
        data={todos}
        renderItem={renderTodoItem}
        keyExtractor={(item) => item.id}
      />
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    padding: 20,
    backgroundColor: "#fff",
    paddingTop: Platform.OS === "android" ?  StatusBar.currentHeight + 10 : 0,
  },
  title: {
    fontSize: 24,
    textAlign: "center",
    fontWeight: "bold",
    marginBottom: 20,
  },
  input: {
    height: 40,
    borderColor: "gray",
    borderWidth: 1,
    borderRadius: 5,
    marginBottom: 10,
    paddingLeft: 10,
  },
  addButton: {
    backgroundColor: "blue",
    padding: 10,
    borderRadius: 5,
    alignItems: "center",
    marginBottom: 10,
  },
  addButtonText: {
    color: "#fff",
    fontSize: 16,
    fontWeight: "bold",
  },
  todoItem: {
    flexDirection: "row",
    justifyContent: "space-between",
    backgroundColor: "#eee",
    padding: 10,
    borderRadius: 5,
    marginBottom: 10,
  },
  deleteButton: {
    color: "red",
  },
});

export default App;
