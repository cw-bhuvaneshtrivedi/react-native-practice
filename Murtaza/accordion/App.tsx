import {
  Platform,
  StatusBar,
  StyleSheet,
  Dimensions,
  Text,
  View,
  TouchableOpacity,
} from "react-native";
import React, { useState } from "react";

import SearchPage from "./src/components/SearchPage";
export default function App() {
  const [click, setClick] = useState(false);
  const handlePress = () => {
    setClick(true);
  };
  return (
    <View style={styles.container}>
      {click ? <SearchPage clicked={setClick} /> : null}
      <View>
        <TouchableOpacity onPress={handlePress} style={styles.page}>
          <View style={styles.button}>
            <Text style={styles.buttonText}>Click Me!</Text>
          </View>
        </TouchableOpacity>
      </View>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: "#efefef",
    // alignItems: 'center',
    // justifyContent: 'center',
    paddingTop: Platform.OS === "android" ? StatusBar.currentHeight + 10 : 0,
  },
  page: { 
    justifyContent: "center", 
    height: Dimensions.get("screen").height },
  button: {
    // flex: 1,
    backgroundColor: "blue",
    padding: 10,
    borderRadius: 5,
    // alignItems: "center",
    justifyContent: "center",
    alignSelf: "center",
  },
  buttonText: {
    color: "white",
    fontSize: 25,
  },
});
