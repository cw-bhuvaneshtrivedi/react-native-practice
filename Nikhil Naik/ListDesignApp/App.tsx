import { StatusBar } from "expo-status-bar";
import { StyleSheet, Text, View } from "react-native";
import Homepage from "./components/Homepage";

export default function App() {
  return (
    <View style={styles.container}>
      <Homepage />
      <StatusBar style="auto" />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: "#232736",
    paddingTop: 35,
  },
});
