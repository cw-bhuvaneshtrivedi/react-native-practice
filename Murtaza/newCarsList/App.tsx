import React from "react";
import {
  StyleSheet,
  View,
  Text,
  TextInput,
  TouchableOpacity,
  FlatList,
  SafeAreaView,
  StatusBar,
  Platform,
  Dimensions,
} from "react-native";
import { FlashList } from "@shopify/flash-list";
import {Card} from "./src/components/Card";
import {data} from "./src/data/Data";
export default function App() {
  return (
    <SafeAreaView style={styles.container}>
        {/* <Text>check...</Text> */}
      <FlashList
        data={data}
        renderItem={({ item }) => <Card item={item} />}
        estimatedItemSize={284}
        getItemType={(item) => item.modelId}
      />
      
    </SafeAreaView>
  );
}

const styles = StyleSheet.create({
  container: {
    // flex: 1,
    backgroundColor: '#fff',
    height: Dimensions.get( 'window' ).height,
    width: Dimensions.get('window').width,
    paddingTop: Platform.OS === "android" ?  StatusBar.currentHeight : 0,
  },
});
