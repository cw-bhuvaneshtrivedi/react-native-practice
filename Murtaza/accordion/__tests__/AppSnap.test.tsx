import React from 'react';
import { render, fireEvent } from '@testing-library/react-native';
import App from '../App';

describe('App', () => {
  test('renders App with search drawer closed correctly', () => {
    const { toJSON } = render(<App />);
    expect(toJSON()).toMatchSnapshot();
  });

  test('renders App with search drawer open correctly', () => {
    const { toJSON, getByText } = render(<App />);
    
    fireEvent.press(getByText('Click Me!'));
    expect(toJSON()).toMatchSnapshot();
  });
});
