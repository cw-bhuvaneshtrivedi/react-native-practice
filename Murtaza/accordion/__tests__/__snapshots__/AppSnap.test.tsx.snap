// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App with search drawer closed correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#efefef",
      "flex": 1,
      "paddingTop": 0,
    }
  }
>
  <View>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "height": 1334,
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <View
        style={
          {
            "alignSelf": "center",
            "backgroundColor": "blue",
            "borderRadius": 5,
            "justifyContent": "center",
            "padding": 10,
          }
        }
      >
        <Text
          style={
            {
              "color": "white",
              "fontSize": 25,
            }
          }
        >
          Click Me!
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`App renders App with search drawer open correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#efefef",
      "flex": 1,
      "paddingTop": 0,
    }
  }
>
  <View
    animatedStyle={
      {
        "value": {
          "transform": [
            {
              "translateY": 1334,
            },
          ],
        },
      }
    }
    collapsable={false}
    style={
      {
        "backgroundColor": "white",
        "height": 1334,
        "marginBottom": 20,
        "transform": [
          {
            "translateY": 1334,
          },
        ],
      }
    }
  >
    <View
      style={
        {
          "height": 100,
          "marginBottom": 20,
          "padding": 16,
        }
      }
    >
      <View
        style={
          [
            {
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
          ]
        }
      >
        <Text
          style={
            {
              "color": "#484848",
              "fontSize": 16,
              "fontWeight": "bold",
            }
          }
        >
          Select Your Make or Model
        </Text>
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "opacity": 1,
            }
          }
          testID="close-button"
        >
          <Text />
        </View>
      </View>
      <TextInput
        placeholder="Type to Select Make "
        style={
          {
            "borderColor": "#d5d5d5",
            "borderRadius": 3,
            "borderStyle": "solid",
            "borderWidth": 2,
            "color": "#aaaaaa",
            "fontSize": 16,
            "height": 44,
            "marginTop": 20,
            "paddingLeft": 16,
          }
        }
      />
    </View>
    <View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": "white",
            "flexDirection": "row",
            "height": 50,
            "justifyContent": "space-between",
            "opacity": 1,
            "paddingLeft": 28,
            "paddingRight": 31,
            "paddingTop": 16,
          }
        }
      >
        <Text
          style={
            {
              "color": "#484848",
              "fontSize": 15,
            }
          }
        >
          Accordion
        </Text>
        <View
          animatedStyle={
            {
              "value": {
                "transform": [
                  {
                    "rotateZ": "0deg",
                  },
                ],
              },
            }
          }
          collapsable={false}
          style={
            {
              "transform": [
                {
                  "rotateZ": "0deg",
                },
              ],
            }
          }
        >
          <Text />
        </View>
      </View>
      <View
        style={
          {
            "backgroundColor": "#efefef",
            "height": 1,
            "width": "100%",
          }
        }
      />
      <View
        animatedStyle={
          {
            "value": {
              "height": 0,
              "overflow": "hidden",
            },
          }
        }
        collapsable={false}
        style={
          {
            "height": 0,
            "overflow": "hidden",
          }
        }
      >
        <View
          onLayout={[Function]}
          style={
            {
              "position": "absolute",
            }
          }
        >
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
    <View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": "white",
            "flexDirection": "row",
            "height": 50,
            "justifyContent": "space-between",
            "opacity": 1,
            "paddingLeft": 28,
            "paddingRight": 31,
            "paddingTop": 16,
          }
        }
      >
        <Text
          style={
            {
              "color": "#484848",
              "fontSize": 15,
            }
          }
        >
          Accordion
        </Text>
        <View
          animatedStyle={
            {
              "value": {
                "transform": [
                  {
                    "rotateZ": "0deg",
                  },
                ],
              },
            }
          }
          collapsable={false}
          style={
            {
              "transform": [
                {
                  "rotateZ": "0deg",
                },
              ],
            }
          }
        >
          <Text />
        </View>
      </View>
      <View
        style={
          {
            "backgroundColor": "#efefef",
            "height": 1,
            "width": "100%",
          }
        }
      />
      <View
        animatedStyle={
          {
            "value": {
              "height": 0,
              "overflow": "hidden",
            },
          }
        }
        collapsable={false}
        style={
          {
            "height": 0,
            "overflow": "hidden",
          }
        }
      >
        <View
          onLayout={[Function]}
          style={
            {
              "position": "absolute",
            }
          }
        >
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
    <View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": "white",
            "flexDirection": "row",
            "height": 50,
            "justifyContent": "space-between",
            "opacity": 1,
            "paddingLeft": 28,
            "paddingRight": 31,
            "paddingTop": 16,
          }
        }
      >
        <Text
          style={
            {
              "color": "#484848",
              "fontSize": 15,
            }
          }
        >
          Accordion
        </Text>
        <View
          animatedStyle={
            {
              "value": {
                "transform": [
                  {
                    "rotateZ": "0deg",
                  },
                ],
              },
            }
          }
          collapsable={false}
          style={
            {
              "transform": [
                {
                  "rotateZ": "0deg",
                },
              ],
            }
          }
        >
          <Text />
        </View>
      </View>
      <View
        style={
          {
            "backgroundColor": "#efefef",
            "height": 1,
            "width": "100%",
          }
        }
      />
      <View
        animatedStyle={
          {
            "value": {
              "height": 0,
              "overflow": "hidden",
            },
          }
        }
        collapsable={false}
        style={
          {
            "height": 0,
            "overflow": "hidden",
          }
        }
      >
        <View
          onLayout={[Function]}
          style={
            {
              "position": "absolute",
            }
          }
        >
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
    <View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "backgroundColor": "white",
            "flexDirection": "row",
            "height": 50,
            "justifyContent": "space-between",
            "opacity": 1,
            "paddingLeft": 28,
            "paddingRight": 31,
            "paddingTop": 16,
          }
        }
      >
        <Text
          style={
            {
              "color": "#484848",
              "fontSize": 15,
            }
          }
        >
          Accordion
        </Text>
        <View
          animatedStyle={
            {
              "value": {
                "transform": [
                  {
                    "rotateZ": "0deg",
                  },
                ],
              },
            }
          }
          collapsable={false}
          style={
            {
              "transform": [
                {
                  "rotateZ": "0deg",
                },
              ],
            }
          }
        >
          <Text />
        </View>
      </View>
      <View
        style={
          {
            "backgroundColor": "#efefef",
            "height": 1,
            "width": "100%",
          }
        }
      />
      <View
        animatedStyle={
          {
            "value": {
              "height": 0,
              "overflow": "hidden",
            },
          }
        }
        collapsable={false}
        style={
          {
            "height": 0,
            "overflow": "hidden",
          }
        }
      >
        <View
          onLayout={[Function]}
          style={
            {
              "position": "absolute",
            }
          }
        >
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "width": 750,
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "#f9f9f9",
                  "height": 50,
                  "paddingBottom": 16,
                  "paddingLeft": 36,
                  "paddingTop": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#484848",
                    "fontSize": 15,
                  }
                }
              >
                AccordionItem
              </Text>
            </View>
            <View
              style={
                {
                  "backgroundColor": "#efefef",
                  "height": 1,
                  "width": "100%",
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
  </View>
  <View>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "height": 1334,
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <View
        style={
          {
            "alignSelf": "center",
            "backgroundColor": "blue",
            "borderRadius": 5,
            "justifyContent": "center",
            "padding": 10,
          }
        }
      >
        <Text
          style={
            {
              "color": "white",
              "fontSize": 25,
            }
          }
        >
          Click Me!
        </Text>
      </View>
    </View>
  </View>
</View>
`;
